src/main/java/org/jabref/model/metadata/FilePreferences.java

```diff
@@ -34,24 +30,14 @@ public String getUser() {
         return user;
     }
 
-    public Optional<Path> getFileDirectory(Field field) {
-        try {
-            String value = fieldFileDirectories.get(field);
-            // filter empty paths
-            if ((value != null) && !value.isEmpty()) {
-                Path path = Paths.get(value);
-                return Optional.of(path);
-            }
-            return Optional.empty();
-        } catch (InvalidPathException ex) {
+    public Optional<Path> getFileDirectory() {
+        if (StringUtil.isBlank(mainFileDirectory)) {
             return Optional.empty();
+        } else {
+            return Optional.of(Paths.get(mainFileDirectory));
```

Comment:
JavaDoc indicates, that `Paths::get` will be deprecated soon. Use `Path::of` instead.
https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/nio/file/Paths.html